src/main/java/com/cloudbees/jenkins/GitHubCommitNotifier.java

```diff
@@ -34,9 +42,16 @@
  */
 public class GitHubCommitNotifier extends Notifier {
 
+    private ExpandableMessage statusMessage = new ExpandableMessage("");
```

Comment:
This doesn't change default behaviour which exists all time.
I can implement token-macro with that logic, but its overhead doing it in this pr i think.
What is your propose?